src/core/store.js

```diff
@@ -1,10 +1,23 @@
-import { applyMiddleware } from 'redux';
+import { applyMiddleware, compose } from 'redux';
 import createLogger from 'redux-logger';
 import config from 'config';
 
-export function middleware({ __config = config } = {}) {
+/*
+ * Enhance a redux store with common middleware.
+ *
+ * This returns a function that takes a single argument, `createStore`,
+ * and returns a new `createStore` function.
+ */
+export function middleware({
+  __config = config, __createLogger = createLogger,
```

Comment:
@mstriemer Any reason to stick with the double underscores here? I only did it for consistency.